lib/nahmii-provider.js

```diff
@@ -147,16 +147,7 @@ class NahmiiProvider extends ethers.providers.JsonRpcProvider {
      * @param {boolean} byAddress - a flag to tell whether to look up by symbol or address
      * @returns {Promise<Object>}
      */
-    async getTokenInfo(symbolOrAddress, byAddress) {
-        if (['ETH', '0X0000000000000000000000000000000000000000'].indexOf(symbolOrAddress.toUpperCase()) !== -1) {
-            return {
-                currency: '0x0000000000000000000000000000000000000000',
-                symbol: 'ETH',
-                decimals: 18,
-                color: ''
-            };
-        }
-
+    async getTokenInfo(symbolOrAddress, byAddress = false) {
```

Comment:
Can we assume the backend will also return the token info for the ETH? If so, then we don't need to mock the ETH token info for convenience anymore.